kentikapi/client_integration_test.go

```diff
@@ -242,15 +269,35 @@ func TestClient_GetAgentWithRetries(t *testing.T) {
 			},
 			expectedResult:  nil,
 			expectedError:   true,
-			errorPredicates: []func(error) bool{kentikapi.IsUnavailableError},
+			errorPredicates: []func(error) bool{kentikapi.IsTemporaryError},
 		}, {
 			name:            "timeout during first request",
 			options:         []kentikapi.ClientOption{kentikapi.WithTimeout(5 * time.Millisecond)},
 			responses:       []gRPCGetAgentResponse{},
 			handlingDelay:   1 * time.Second,
 			expectedResult:  nil,
 			expectedError:   true,
-			errorPredicates: []func(error) bool{kentikapi.IsTimeoutError},
+			errorPredicates: []func(error) bool{kentikapi.IsTimeoutError, kentikapi.IsTemporaryError},
+		}, {
+			name: "authorization error",
```

Comment:
```suggestion
			name: "Unauthenticated error",
```
authorization is not equal authentication (but we use AuthError for both)

Reply:
Done